test/TestExpectations.json

```diff
@@ -1080,6 +1080,54 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["FAIL"]
   },
+  {
```

Comment:
Could you please add a `"comment":""` field with a comment about the reason for the fail expectation?

Reply:
done